tests/plots/test_declarative.py

```diff
@@ -2133,6 +2133,42 @@ def test_declarative_plot_geometry_lines(ccrs):
     return pc.figure
 
 
+@pytest.mark.mpl_image_compare(remove_text=False, tolerance=2.985)
```

Comment:
Is this high of a tolerance really needed?

Reply:
No. That is what we call a copy and paste and don't think about it error.

Reply:
😆